src/main/php/scriptlet/xml/workflow/Wrapper.class.php

```diff
@@ -231,7 +231,10 @@ public function load($request, $handler) {
       // the string "multiple", the array will be preserved. Otherwise, the
       // first element will be copied to the values hash, thus making 
       // accessibility easy.
-      if (empty($value) || 0 == strlen(implode($value)) || self::isEmptyFileUpload($value)) {
+      if (
+        (self::isFileUpload($value) && self::isEmptyFileUpload($value)) ||
+        (!self::isFileUpload($value) && (empty($value) || 0 == strlen(implode($value))))
```

Comment:
The `implode()` part here is what causes the issue: `implode($value);` with $value being a multi-dimensional array produces a E_WARNING which contains the necessary bytes for the XP error handler to trigger a ClassCastException(!). In XP5 this code worked w/o an exception.